pages/_app.js

```diff
@@ -14,7 +16,7 @@ const App = ({ Component }) => {
             window.pool = new Pool(`${window.location.protocol === 'http:' ? 'ws' : 'wss'}://${window.location.host}`);
             setPool(pool);
```

Comment:
I think pool will always be null but maybe I'm missing something ?

Reply:
Yes didn't see in first PR. Nice catch.

In future PR, the code will be remove from the `_app.js` file.